dm: update doc for error handling (#7894)
pingcap/docs: dm/dmctl-introduction.md

  dmctl [command]

Available Commands:

>   binlog          manage or show binlog operations
>   binlog-schema   manage or show table schema in schema tracker
>   check-task      Checks the configuration file of the task
>   config          manage config operations

Reviewer: ```suggestion
  binlog          Manages or shows binlog operations
  binlog-schema   Manages or shows the table schema in schema tracker
  check-task      Checks the configuration file of the task
  config          Manages config operations
```

```
Available Commands:

>   binlog          manage or show binlog operations
>   binlog-schema   manage or show table schema in schema tracker
>   check-task      Checks the configuration file of the task
>   config          manage config operations

Reviewer: ```suggestion
  binlog          Manages or shows binlog operations
  binlog-schema   Manages or shows the table schema in schema tracker
  check-task      Checks the configuration file of the task
  config          Manages config operations
```

  decrypt         Decrypts cipher text to plain text
  encrypt         Encrypts plain text to cipher text
  help            Gets help about any command
  list-member     Lists member information
  offline-member  Offlines member which has been closed
  operate-leader  `evict`/`cancel-evict` the leader
  operate-source  `create`/`update`/`stop`/`show` upstream MySQL/MariaDB source
  pause-relay     Pauses DM-worker's relay unit

>   pause-task      Pauses a specified running task or all (sub)tasks bound to a source

Reviewer: ```suggestion
  pause-task      Pauses a specified running task or all tasks or subtasks bound to a source
```

  purge-relay     Purges relay log files of the DM-worker according to the specified filename
  query-status    Queries task status
  resume-relay    Resumes DM-worker's relay unit

>   resume-task     Resumes a specified paused task or all (sub)tasks bound to a source

Reviewer: ```suggestion
  resume-task     Resumes a specified paused task or all tasks or subtasks bound to a source
```

  shard-ddl-lock  maintain or show shard-ddl locks information
  start-relay     Starts workers pulling relay log for a source
  start-task      Starts a task as defined in the configuration file
  stop-relay      Stops workers pulling relay log for a source

>   stop-task       Stops a specified task or all (sub)tasks bound to a source

Reviewer: ```suggestion
  stop-task       Stops a specified task or all tasks or subtasks bound to a source
```
